Makefile

```diff
@@ -6,9 +6,12 @@ all: build docker
 
 .PHONY: build docker
 
-build:
+build: test
 	GOOS=linux go build -v -o ./bin/${BINARY_NAME} ./cmd/main.go
 
+test:
+	go test -v ./cfg... ./pkg... ./utils...
+
```

Comment:
Consider also adding a `ci` target to run `golangci-lint` as we have for the plugin broker: [Makefile](https://github.com/eclipse/che-plugin-broker/blob/ac7f5c3fb78593444bd634be72b4ed42451e3f6b/Makefile#L10) and [Dockerfile](https://github.com/eclipse/che-plugin-broker/blob/master/build/CI/Dockerfile)

Reply:
awesome, I will add that